vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import react from "@vitejs/plugin-react";
+import {defineConfig} from "vite";
+
+
+// https://vite.dev/config/
+export default defineConfig({
+    appType: "mpa",
```

Comment:
if with default "spa", anything that doesn't exist would be redirected to the index.html file. e.g.,
`http://localhost:3010/test/non-exist.clp.zst` -> ``http://localhost:3010/index.html`